models/property.py

```diff
@@ -61,4 +62,14 @@ def find_by_name(cls, name):
 
     @classmethod
     def find_by_manager(cls, manager_id):
-        return cls.query.filter_by(propertyManager=manager_id).all()
+        return cls.query.filter(cls.managers.any(UserModel.id == manager_id)).all()
+
+    @classmethod
+    def set_property_managers(cls, ids):
```

Comment:
Should we create a unit test for this method?

Reply:
I can add a test for it.